doc/api/async_storage.md

```diff
@@ -0,0 +1,86 @@
+# Async Storage
+
+<!--introduced_in=v12.x.x-->
```

Comment:
is this supposed to be `added: REPLACEME` ? Does `introduced_in` work also?

Reply:
I think it’s a separate system used for the per-page “View another version” dropdown in the generated documentation:

![image](https://user-images.githubusercontent.com/899444/54071214-e86d7e80-4269-11e9-8be9-2703b9c1efc5.png)